src/service.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import DeclarativeBase
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+URL_DATABASE = (
+    f"postgresql://{os.environ.get("POSTGRES_USER")}:"
+    f"{os.environ.get("POSTGRES_PASSWORD")}@db:5432/{os.environ.get("POSTGRES_DATABASE")}"
+)
```

Comment:
same as previous comment related to db creds.
additionally you could move such things to some sort of config file, let's say config.py or settings.py. But here is absolutelly fine. But. before you use them in the f string worth assigning to the appropriate var, e.g.
POSTGRES_USER = os.environ.get("POSTGRES_USER")